pinakes/main/inventory/serializers.py

```diff
@@ -35,6 +41,63 @@ class Meta:
         )
         read_only_fields = ("created_at", "updated_at")
 
+    def get_refresh_state(self, obj):
+        return _(obj.refresh_state)
+
+    def get_last_refresh_message(self, obj):
+        if obj.availability_status == "unavailable":
+            return _("%(name)s is unavailable, refresh skipped") % {
+                "name": obj.name
+            }
+
+        if obj.refresh_state == Source.State.FAILED:
+            return _("Refresh faild: %(error)s") % {
+                "error": obj.last_refresh_message
+            }
+
+        sii_stats = obj.last_refresh_stats.get("service_inventory", {})
+        soi_stats = obj.last_refresh_stats.get("service_offering", {})
+        son_stats = obj.last_refresh_stats.get("service_offering_node", {})
+
```

Comment:
@hsong-rh
Maybe we can just add a gextnext_noop
https://docs.djangoproject.com/en/4.0/topics/i18n/translation/#marking-strings-as-no-op

gettext_noop("add")
gettext_noop("deletes")
gettext_noop("updates")

The above lines will allow us to extract the strings out which can be used as _(key) when creating the dicts below